src/js/helpers/covid19RequestsHelper.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * covid19RequestsHelper.js
+ * Created By Lizzie Salita 6/10/20
+ */
+
+import { apiRequest } from './apiRequest';
+
+export const fetchOverview = () => apiRequest({
+    isMocked: true,
+    url: 'v2/disaster/overview/'
```

Comment:
Can we remove this trailing slash? Is there a reason why we're doing this?

Reply:
We're use trailing slashes at the end of all our api request urls. (Also matches the convention used in API contracts)